package.json

```diff
@@ -33,21 +36,13 @@
     "truffle-hdwallet-provider": "0.0.3"
   },
   "devDependencies": {
-    "eslint-plugin-import": "2.7.0",
+    "eslint": "4.13.1",
+    "eslint-config-airbnb": "16.1.0",
+    "eslint-plugin-import": "2.8.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
```

Comment:
Can we remove jsx-a11y? I don't think this repo has any jsx in it?

Reply:
for whatever reason, when I remove it, stuff breaks

Reply:
Okay, I won't block on that. Just pin the dependency and add an issue to investigate it later.